modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/rest-openapi.yaml

```diff
@@ -78,6 +78,23 @@ components:
                     readOnly: true
                     type: string
             type: object
+        Node:
```

Comment:
Hey @MarinhoFeliphe , please add the @review flag to the technical writers be aware of this text. Thx